qa/suites/rgw/multisite/all/multi_cluster.yaml

```diff
@@ -0,0 +1,84 @@
+meta:
+- desc: run two ceph clusters 
+roles:
+- - cluster1.mon.a
+  - cluster1.mon.b
+  - cluster1.osd.0
+  - cluster1.osd.1
+  - cluster1.osd.2
+  - cluster2.mon.c
+  - cluster1.client.0
+  - cluster2.client.0
+  - cluster1.mgr.x
+- - cluster1.mon.c
+  - cluster2.mon.a
+  - cluster2.mon.b
+  - cluster2.osd.0
+  - cluster2.osd.1
+  - cluster2.osd.2
+  - cluster2.mgr.y
+tasks:
+- install:
+- ceph:
+    cluster: cluster1
+    conf:
+      client.0:
+        rgw region: jerry
+        rgw zone: seinfeld
+        rgw region root pool: .rgw.region.0
```

Comment:
We should use `rgw zonegroup root pool ` and not `rgw region root pool`
Also what about` rgw period root pool` and `rgw realm root pool`?